### PR TITLE
Fix fee breakdown

### PR DESCRIPTION
**Pull request by ajchilts**
Fix to fee breakdown table for appropriate business types given answers to F&P and Approval. 

## Related / Dependant PRs?

N/A

## How Has This Been Tested?

Unit and Local Testing

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.

### CODE REVIEW

File: app/typeclasses/confirmation/ResponsiblePeopleRows.scala [hunk 1]
```diff
@@ -52,25 +53,28 @@ trait ResponsiblePeopleRows[A] extends FeeCalculations {
                                                         ): Seq[BreakdownRow] = {
 
     val (notPassedFP, notPassedApprovalCheck) = (value.addedResponsiblePeopleFitAndProper, value.addedResponsiblePeopleApprovalCheck)
+    val msbOrTcsp = (activities.contains(MSB) || activities.contains(TrustAndCompanyServices))
+    val bpsTdi = (activities.contains(BPS) || activities.contains(TPS))
```
Comment (slavvco) [inline app/typeclasses/confirmation/ResponsiblePeopleRows.scala]: if you like you could extract above two into defs and put them in BusinessActivities object, then reuse them:
def msbOrTcsp(activities: Set[BusinessActivity]) = activities.contains(MSB) || activities.contains(TrustAndCompanyServices)
  def bpsTdi(activities: Set[BusinessActivity]) = activities.contains(BPS) || activities.contains(TPS)

File: app/typeclasses/confirmation/ResponsiblePeopleRows.scala [hunk 2]
```diff
@@ -52,25 +53,28 @@ trait ResponsiblePeopleRows[A] extends FeeCalculations {
...
 
-    (notPassedFP > 0, notPassedApprovalCheck > 0) match {
-      case (true, _) => Seq(
-        BreakdownRow(
-          peopleRow(value).message,
-          value.addedResponsiblePeopleFitAndProper,
-          peopleRow(value).feePer,
-          Currency.fromBD(value.getFpFee.getOrElse(0))
-        ))
+    (notPassedFP > 0, notPassedApprovalCheck > 0, msbOrTcsp, bpsTdi) match {
```
Comment (slavvco) [inline app/typeclasses/confirmation/ResponsiblePeopleRows.scala]: then here:
msbOrTcsp(activities), bpsTdi(activities)

File: app/typeclasses/confirmation/ResponsiblePeopleRows.scala [hunk 3]
```diff
@@ -81,30 +85,28 @@ trait ResponsiblePeopleRows[A] extends FeeCalculations {
...
-            fitAndProperCount,
-            peopleRow(value).feePer,
-            Currency.fromBD(value.getFpFee.getOrElse(0))
-          ))
-
-      case _ => Seq.empty
+    val msbOrTcsp = (activities.contains(MSB) || activities.contains(TrustAndCompanyServices))
+    val bpsTdi = (activities.contains(BPS) || activities.contains(TPS))
+
+    (fitAndProperCount > 0, approvalCheckCount > 0, msbOrTcsp, bpsTdi) match {
```
Comment (slavvco) [inline app/typeclasses/confirmation/ResponsiblePeopleRows.scala]: and here:
msbOrTcsp(activities), bpsTdi(activities)

**Comment by ajchilts** [comment]
Closing and re-target `phase-2-release-candidate-rc1`

**Comment by ajchilts** [comment]
Mike Pratt added a comment - Yesterday
Retested & all passed.

**Pull request merged**